numpy/__init__.pxd

```diff
@@ -601,7 +601,7 @@ cdef extern from "numpy/arrayobject.h":
     int PyArray_Sort (ndarray, int, NPY_SORTKIND)
     object PyArray_ArgSort (ndarray, int, NPY_SORTKIND)
     object PyArray_SearchSorted (ndarray, object, NPY_SEARCHSIDE, PyObject *)
-    object PyArray_ArgMax (ndarray, int, ndarray)
+    object PyArray_ArgMax (ndarray, int, ndarray, int)
```

Comment:
This is a breaking change to the numpy C API - you'll need to add a new `PyArray_ArgMinEx` or similar to prevent breaking downstream code.

Reply:
Is there a possibility to overload it?